common/StringVector.cpp

```diff
@@ -127,4 +127,11 @@ StringVector StringVector::tokenizeAnyOf(const std::string& s, const char* delim
     return StringVector(s, std::move(tokens));
 }
 
+bool StringVector::getIntParam(std::size_t index, int &value) const
```

Comment:
Is there really no way to get an integer parameter from a StringVector already ?
Can you see how the existing method:
    bool getUInt32(std::size_t index, const std::string& key, uint32_t& value) const;
Works - and see how other protocol messages are structured, and re-use the same key/value schema ?

Reply:
` bool getUInt32(std::size_t index, const std::string& key, uint32_t& value) const;`
can't be used here given the input format. It can only parse "key intvalue" or "key=intvalue".
But here I have currently "key value value value".
The format can be changed to follow to more conventional: 
```
key1 intvalue
key2 intvalue
...
```
I was using this pattern before sending everything in a single message.